### PR TITLE
Provide more clarity to the QA test readme

### PR DESCRIPTION
**Pull request by purbon**
Try to clarify the workflow to run the acceptance test as of now.

@jsvd what do you think?

### CODE REVIEW

File: qa/README.md [hunk 1]
```diff
@@ -60,19 +59,20 @@ rake qa:acceptance:redhat           # Run acceptance test in redhat machines
...
-platform you should use
+before you *will have to bootstrap* all selected machines, you can do
+that using the `rake qa:vm:setup[platform]` task. This is done like this
+as bootstrap imply setting up the VM'S and this might take some time and
+you might only want to this once.
+
+In the feature we might add new rake tasks to do all at once, but for now you can use the script under
+`ci/ci_acceptance.sh` to do all at once.
+
+For local testing puroposes, is recommended to not run all together, pick your target and run with the single platform command, If you're willing to run on single platform you should use:
```
Comment (jsvd) [inline qa/README.md]: the command you list below does not run a single platform but a single machine.

File: qa/README.md [hunk 2]
```diff
@@ -81,6 +81,9 @@ file, this could be done running `rake qa:vm:ssh_config`. When this task
 is finished a file named `.vm_ssh_config` will be generated with all the
 necessary information to connect with the different machines.
 
+If you are *running this test for first time*, you will need to setup
```
Comment (jsvd) [inline qa/README.md]: I'd put this sentence as the first one of this `How to run tests` chapter

**Comment by suyograo** [comment]
Any updates to this?

**Comment by suyograo** [comment]
@purbon update ping?

**Comment by purbon** [comment]
@jsvd fixed your comments, let me know what do you think?

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**